JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/Services/AuthenticationService.cs

```diff
@@ -11,7 +11,7 @@ public class AuthenticationService : IAuthenticationService
         private const string AuthenticationUrl = "api/authenticate";
         private const string AccountUrl = "api/account";
         private const string AuthorizationHeader = "Authorization";
-        private readonly HttpClient _httpClient;
+        public HttpClient _httpClient { get; set; }
```

Comment:
httpClient is already an injected singleton with mvvmcross you never need to make it public

Reply:
He's not injected, look at the test. I need to do like this. I don't know of any other way.